src/blob-store/index.js

```diff
@@ -274,6 +277,55 @@ class PretendCorestore {
     }
   }
 
+  /**
+   * Write blobs for provided variants of a file
+   * @param {{ original: string, preview?: string, thumbnail?: string }} filepaths 
+   * @param {{ mimeType?: string }} metadata
+   * @returns {Promise<{ original: Omit<BlobId, 'driveId'>, preview?: Omit<BlobId, 'driveId'>, thumbnail?: Omit<BlobId, 'driveId'> }>}
+   */
+  async create(filepaths, metadata) {
```

Comment:
[Client docs](https://github.com/digidem/mapeo-core-docs/blob/main/client-docs/README.md#blobcreate) specify that `metadata` is an optional arg:

```suggestion
   * @param {{ mimeType?: string }} [metadata={}]
   * @returns {Promise<{ original: Omit<BlobId, 'driveId'>, preview?: Omit<BlobId, 'driveId'>, thumbnail?: Omit<BlobId, 'driveId'> }>}
   */
  async create(filepaths, metadata = {}) {
```

Reply:
as in `@param {{ mimeType: string }} metadata`?